recipes/libcurl/all/conanfile.py

```diff
@@ -207,13 +205,24 @@ def build_requirements(self):
         else:
             self.build_requires("libtool/2.4.6")
             self.build_requires("pkgconf/1.7.4")
```

Comment:
```suggestion
            self.tool_requires("libtool/2.4.6")
            self.tool_requires("pkgconf/1.7.4")
```

and same for ninja which I can't review here